src/sentry/profiles/flamegraph.py

```diff
@@ -23,7 +24,7 @@ def query_profiles_data(
         params=params,
         query=query,
         selected_columns=selected_columns,
-        limit=200,
+        limit=options.get("profiling.flamegraph.profile-set.size"),
```

Comment:
This option has a default of 0. It may be good to have some kind of fall back like `options.get("profiling.flamegraph.profile-set.size") or 200` (perhaps a log message), so that we never accidentally try to aggregate 0 profiles.